app/components/pipeline/Header/index.js

```diff
@@ -189,7 +174,7 @@ class Header extends React.Component {
     if (this.props.pipeline.description) {
       return <Emojify text={this.props.pipeline.description} />;
     } else {
-      return this.repositoryUrl();
+      return this.props.pipeline.repository.url;
```

Comment:
We should probably improve the documentation of those GraphQL fields, because I think that wouldn't have helped with the confusion at all! I'll do that now.